models/platformRoles.js

```diff
@@ -6,6 +6,7 @@ module.exports = {
     createdBy : String,
     updatedBy : String,
     status : String,
-    isDeleted: Boolean
+    isDeleted: Boolean,
```

Comment:
@rakeshSgr - Let's add proper validations like default value, also code and title, etc cannot be blank

Reply:
this is an already created, model. I will update the changes.

Reply:
what is the default values for code and title ?